esp/esp/local_settings.py.travis

```diff
@@ -46,6 +46,7 @@ Learning Unlimited, Inc.
 
 SITE_INFO = (1, 'testserver.learningu.org', 'ESP Test Server')
 CACHE_PREFIX = "Test"
+SECRET_KEY = 'b36cfdfb78aba27ddbde330b23352bc6d40973b4'
```

Comment:
Good catch. Fabric/Vagrant dev servers already have it, including old ones, but we'll need to make sure the deploy process for production sites includes adding one, and add it to whatever script we're using these days to create new sites. And all you people still using legacy dev setup will have to fix it on your dev servers.